torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -932,10 +962,11 @@ def perspective_image_tensor(
     fill: features.FillTypeJIT = None,
     coefficients: Optional[List[float]] = None,
 ) -> torch.Tensor:
-    perspective_coeffs = _perspective_coefficients(startpoints, endpoints, coefficients)
```

Comment:
In a previous PR, I was asked to move checks on top. This is a check and an estimation, but regardless this is why I put it there. No strong opinion but if that's a perf opt, we optimize for a corner-case that we don't care too much (0 batch size).

Reply:
OK, I can revert this change.